src/SFA.DAS.RoatpGateway.Web/ModelBinders/StringTrimAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace SFA.DAS.RoatpGateway.Web.ModelBinders
+{
+    /// <summary>
```

Comment:
you don't need all these comments

Reply:
It's a public attribute so I would say the opposite such that the person using it knows what it does and the limitations